packages/cli/lib/project-generator.js

```diff
@@ -246,13 +246,4 @@ module.exports = class ProjectGenerator extends BaseGenerator {
       this.fs.delete(this.destinationPath('.vscode'));
     }
   }
-
-  install() {
-    if (this.shouldExit()) return false;
```

Comment:
don't we need to explicitly call `await super.install()` here? I don't think it's going to be run otherwise.
This brings up the question of whether wanting a test to see whether the dependencies have been physically installed or not. Seems like a bit of a hassle though

Reply:
Good question and I don't think so -- I should probably return `super.install()` though. If you look at the output in the PR description, the project dependencies are installed before there is output printed to the console.